.coveragerc

```diff
@@ -1,2 +1,2 @@
 [run]
-omit = */tests/*, */scripts/*, **/*.jinja, **/*.html, **/*.txt
+omit = */tests/*, */scripts/*, **/*.jinja, **/*.html, **/*.txt, flexmeasures/data/migrations/versions/*
```

Comment:
@nhoening I've changed the `.coveragerc` file in this pull request to exclude the db migration files. Does this need a changelog entry? 

The main goal of this pull request is to make sure the models and db are lined up again but I want to make sure this change does not slip by without you knowing about it.